### DISCUSSION TITLE
[Feature] Merge people into single identity

### DISCUSSION THREAD

**Discussion by Oreoezi** [immich-app/immich#11372] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The AI face-recognition created two different people for the same person. I would like, in that case, to be manually able to group them together. 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
When viewing a person, there's a merge option in the top right.
